Cargo.toml

```diff
@@ -17,7 +17,8 @@ features     = ["docs"]
 rustdoc-args = ["--cfg", "feature=\"docs\""]
 
 [features]
-default = ["builder"]
+default = ["builder", "reqwest/default-tls", "reqwest/json"]
```

Comment:
Add a new native-tls feature and include reqwest/default-tls as dependency, then include it in default.